### PR TITLE
BorrowTable id converted to class

### PR DESCRIPTION
**Pull request by ArunTeltia**
Closes #

### Technical

There are two file that have id at two places so for fixing that i conveerted all the places of borrowtable id into class

### Stakeholders

@jamesachamp @jdlrobson

### CODE REVIEW

File: openlibrary/templates/account/borrow.html [hunk 1]
```diff
@@ -164,7 +164,7 @@ <h1>$_("Books You're Waiting For")</h1>
         });
     });
     </script>
-    <div id="borrowTable" class="borrow collapse waitinglist">
+    <div class="borrow borrowTable collapse waitinglist">
```
Comment (jdlrobson) [inline openlibrary/templates/account/borrow.html:167]: could the existing borrow class be used?
Does borrow mean something else, elsewhere?

If not, I suggest using a new class borrow-table which is more BEM-like : http://getbem.com/naming/

**Review by jdlrobson — CHANGES_REQUESTED** [review]
You'll need to run `npm test` to address the linting issues.

File: openlibrary/templates/account/borrow.html (hunk 1 above)
Comment (ArunTeltia) [reply openlibrary/templates/account/borrow.html:167]: I tried checking by vscode search option and I was not able to find any more instance of borrowtable in any file other than what I changed 

Will change it to borrow-table to make it bem like

**Comment by ArunTeltia** [comment]
Okay I will do that

**Review by jdlrobson — APPROVED** [review]
LGTM

File: static/css/page-plain.less [hunk 2]
```diff
@@ -38,10 +35,49 @@ div.nav {
   margin: 50px 0;
   font-size: 14px;
 }
+// Specific to borrowTable component
+
+.borrow-table {
```
[the remark this answers is not in the document]
Comment (ArunTeltia) [reply static/css/page-plain.less:40]: Yes sir I have noted actually i learned about this from one of the pr in which u guided koderjoker on how to do it thats how i resolve the issue, and i will remember this in future

**Pull request merged**